src/overlays/actors/ovl_Bg_Spot16_Bombstone/z_bg_spot16_bombstone.c

```diff
@@ -21,49 +109,447 @@ const ActorInit Bg_Spot16_Bombstone_InitVars = {
+
+        if (adjustedYawDiff > 0) {
+            sinValue = Math_Sins(adjustedYawDiff) * this->actor.xzDistFromLink;
+
+            if (sinValue >= 0.0f) {
+                player->actor.posRot.pos.x += sinValue * this->sinRotation;
+                player->actor.posRot.pos.z += sinValue * this->cosRotation;
+            } else {
+                osSyncPrintf("Error 補正出来ない(%s %d)(arg_data 0x%04x)(hosei_angY %x)\n",
+                             "../z_bg_spot16_bombstone.c", 935, this->actor.params, adjustedYawDiff);
+            }
+        }
+    }
+}
+
+void func_808B57E0(BgSpot16Bombstone* this, GlobalContext* globalCtx) {
+    EnBombf* playerHeldActor;
+    Player* player = PLAYER;
+    EnBombf* currentBomb;
+
+    if (timer > 0) {
+        timer--;
+    }
+
+    if (playerBomb != NULL) {
+        if (playerBomb->actor.update == NULL) {
+            playerBomb = NULL;
+        } else if (timer <= 0 && playerBomb->actor.posRot.pos.y < 1400.0f &&
+                   Math3D_Dist1DSq(playerBomb->actor.posRot.pos.x + 1579.0f, playerBomb->actor.posRot.pos.z + 790.0f) <
+                       160000.0f &&
+                   playerBomb->actor.params == 0) {
+            currentBomb = playerBomb;
+            if (currentBomb->timer > 0) {
+                timer = currentBomb->timer + 0x14;
+                func_800800F8(globalCtx, 0x1054, timer, NULL, 0);
+            }
+        }
+    } else {
+        if ((player->stateFlags1 & 0x800) != 0) {
+            playerHeldActor = player->heldActor;
+            if (playerHeldActor != NULL && playerHeldActor->actor.type == ACTORTYPE_EXPLOSIVES &&
+                playerHeldActor->actor.id == ACTOR_EN_BOMBF) {
+                playerBomb = playerHeldActor;
+            }
+        }
+    }
+}
+
+void func_808B5934(BgSpot16Bombstone* this) {
+    this->actor.draw = &BgSpot16Bombstone_Draw;
+    this->actionFunc = &func_808B5950;
+}
+
+void func_808B5950(BgSpot16Bombstone* this, GlobalContext* globalCtx) {
+    u8 acFlags;
+
+    func_808B56BC(this, globalCtx);
+    func_808B57E0(this, globalCtx);
+
+    if (globalCtx) {}
+
+    acFlags = this->colliderCylinder.base.acFlags;
+
+    if ((acFlags & 2) != 0) {
+        this->colliderCylinder.base.acFlags = acFlags & 0xFFFD;
+
+        func_808B561C(this, globalCtx);
+
+        func_800800F8(globalCtx, 0x1054, 0x32, NULL, 0);
+
+        Flags_SetSwitch(globalCtx, this->switchFlag);
+        gSaveContext.eventChkInf[2] |= 8;
+
+        func_808B5A78(this);
+    } else {
+        CollisionCheck_SetAC(globalCtx, &globalCtx->colChkCtx, &this->colliderCylinder);
+        CollisionCheck_SetOC(globalCtx, &globalCtx->colChkCtx, &this->colliderJntSph);
+        CollisionCheck_SetAC(globalCtx, &globalCtx->colChkCtx, &this->colliderJntSph);
+    }
+
+    if (gGameInfo->data[0x8e0] == 1) {
+        func_808B561C(this, globalCtx);
+        gGameInfo->data[0x8e0] = -0xA;
+    } else if (gGameInfo->data[0x8E0] < 0) {
+        gGameInfo->data[0x8E0]++;
+    }
```

Comment:
You can use `./tools/regconvert.py 0x8e0` to get the macro for accessing `gGameInfo->data[0x8E0]`
Looks like it's a timer (?) so maybe `-0xA` -> `-10`

Reply:
I gave that a try and it gave me "CREG(70)" as the output.  I'm not sure how to use that.  I tried putting it in place of gGameInfo->data[0x8e0] and just 0x8e0 and both resulted in not OK.

Reply:
My bad, the script takes the offset relative to `gGameInfo + 0`, which is `0x8E0 * sizeof(s16) + OFFSETOF(GameInfo, data) = 0x8E0 * 2 + 0x14 = 0x11d4`

try `./tools/regconvert.py 0x11d4` 😅 
it should replace `gGameInfo->data[0x8e0]`